### PR TITLE
another npm update && fix of red incompatible background

### PR DESCRIPTION
**Pull request by knoxfighter**
Updated the npm packages again.
Also i fixed the bug, that the incompatiblity of mods were not shown, when downloading by mod-ID.

Updates:
- @babel/core ~~7.0.0~~ -> 7.2.2
- @babel/preset-env ~~7.0.0~~ -> 7.2.3
- @fortawesome/fontawesome-free ~~5.2.0~~ -> 5.6.3
- babel-loader ~~8.0.0~~ -> 8.0.5
- bootstrap ~~4.1.3~~ -> 4.2.1
- bootstrap-fileinput ~~4.4.9~~ -> 4.5.2
- css-loader ~~1.0.0~~ -> 2.1.0
- file-loader ~~2.0.0~~ -> 3.0.1
- mini-css-extract-plugin ~~0.4.1~~ -> 0.5.0
- node-sass ~~4.9.3~~ -> 4.11.0
- react ~~16.4.2~~ -> 4.11.0
- react-dom ~~16.4.2~~ -> 16.7.0
- resolve-url-loader ~~2.3.0~~ -> 3.0.0
- semver ~~5.5.0~~ -> 5.6.0
- sweetalert2 ~~7.26.11~~ -> 7.33.1
- webpack ~~4.16.5~~ -> 4.28.4
- webpack-cli ~~3.1.0~~ -> 3.2.1
- webpack-fix-style-only-entries ~~0.0.1~~ -> 0.1.0

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -61,7 +61,6 @@ module.exports = (env, argv) => {
                             loader: "css-loader",
                             options: {
                                 "sourceMap": !isProduction,
-                                "minimize": isProduction
```
Comment (mroote) [inline webpack.config.js]: Do we want to drop the minimization?

Comment (knoxfighter) [reply webpack.config.js]: The minimization feature of the css-loader got dropped in an earlier version. It only was not breaking the chain, it was just ignored. The output in production mode is still minimized, even without this configuration.

Comment (mroote) [reply webpack.config.js]: Ah cool sounds good.

**Comment by mroote** [comment]
Thanks!

**Pull request merged**